### ISSUE TITLE
[addition] add life series members

### ISSUE THREAD

**Issue by UplandJacob** [dreadhalor/hermitcraft-horns#14]
There are plenty of horns to be made from non-hermit life series members.
Possibly have a section for non-hermits when selecting hermit for a horn

Bigbst4tz2
InTheLittleWood
LDShadowLady
Smajor1995
Solidarity

**Comment by dreadhalor** [comment]
Good idea - I haven't touched this codebase in a while but I can add that as a priority when I get back to it
